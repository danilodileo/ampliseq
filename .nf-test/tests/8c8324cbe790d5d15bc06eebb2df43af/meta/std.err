curl: (77) error setting certificate file: /ssl/cacert.pem
