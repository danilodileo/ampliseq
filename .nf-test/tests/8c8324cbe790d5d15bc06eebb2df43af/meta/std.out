ERROR ~ No such file or directory: Can't find a matching module file for include: ../../../modules/nf-core/mafft/main

 -- Check script '/Users/dadiaa/projects/dev/ampliseq/./workflows/../subworkflows/nf-core/fasta_newick_epang_gappa/main.nf' at line: 9 or see '/Users/dadiaa/projects/dev/ampliseq/.nf-test/tests/8c8324cbe790d5d15bc06eebb2df43af/meta/nextflow.log' file for more details
